Content.Client/SurveillanceCamera/UI/SurveillanceCameraMonitorWindow.xaml.cs

```diff
@@ -71,15 +74,23 @@ public SurveillanceCameraMonitorWindow()
             SubnetOpened!((string) args.Button.GetItemMetadata(args.Id)!);
         };
         SubnetRefreshButton.OnPressed += _ => SubnetRefresh!();
+        SubnetRefreshButtonMap.OnPressed += _ => SubnetRefresh!();
         CameraRefreshButton.OnPressed += _ => CameraRefresh!();
         CameraDisconnectButton.OnPressed += _ => CameraDisconnect!();
```

Comment:
This isn't your fault, but every single event incoation in this file should be doing `?.Invoke()` instead of null supressing it. Otherwise this causes a crash if you have something initalize this window and not subscribe to one of those events.